spec/abci++/v0.md

```diff
@@ -0,0 +1,161 @@
+# Tendermint v0 Markdown pseudocode
+
+This translates the latex code for Tendermint consensus from the Tendermint paper into markdown.
+
+### Initialization
+
+```
+h_p = 0
+round_p = 0
+step_p is one of {propose, prevote, precommit}
+decision_p = nil
+lockedValue_p = nil
+validValue_p = nil
+validRound_p = -1
+```
+
+### StartRound(round)
+
+```go
+function startRound(round) {
+    round_p ← round
+    step_p ← propose
+    if proposer(h_p, round_p) = p {
+        if validValue_p != nil {
+            proposal ← validValue_p
+        }
+        else {
+            proposal ← getValue()
```

Comment:
Again, orthogonal to this PR:
The same holds for the bft paper but `getvalue()` is not introduced or explained anywhere. This if-else-block could potenially be simplified / remove by explaining getValue.

Reply:
OK, that makes sense.